bigquery/api/Snippets/LoadFromFile.cs

```diff
@@ -32,7 +32,7 @@ public class BigQueryLoadFromFile
         var uploadCsvOptions = new UploadCsvOptions()
         {
             SkipLeadingRows = 1,  // Skips the file headers
-            Autodetect = true
+            Autodetect = true // Set to false if the table already exists
```

Comment:
This is not entirely accurate, or may be for this particular piece of code, but not in general.
See here in the [docs](https://cloud.google.com/bigquery/docs/reference/rest/v2/Job#JobConfigurationLoad) how this setting interacts with other settings, in particular `schemaUpdateOptions` and even `skipLeadingRows`.
Happy to accept a more general comment here, or something that links to the documentation.